dominion-pax-exam-driver/src/main/java/org/codice/dominion/pax/exam/internal/DominionKarafConfigurationFile.java

```diff
@@ -16,52 +16,79 @@
 import java.io.File;
 import java.io.IOException;
 import javax.annotation.Nullable;
+import org.apache.commons.io.FileUtils;
+
+/** Base class for Karaf configuration file support. */
+public abstract class DominionKarafConfigurationFile {
+  protected final File file;
+
+  public DominionKarafConfigurationFile(final File karafHome, final String location) {
```

Comment:
That code already handles it automatically. This was copied over from PaxExam to allow us to support additional operations. The other places where separators are converted is mainly because the expectation of all PaxExam options (and out annotations) is that relative path names are always done using"/" and PaxExam converts it wall appropriately in the end.